roles/ensure_machineset/tasks/derive_machineset.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Get the names of an existing worker machinesets (of any instance type)
+  command:
+    oc get machinesets -n openshift-machine-api -o
+    jsonpath='{range .items[?(@.spec.template.metadata.labels.machine\.openshift\.io/cluster-api-machine-role=="worker")]}{.metadata.name}{"\n"}{end}'
+  register: oc_get_machinesets
+  failed_when: not oc_get_machinesets.stdout
+
+- name: Store the MachineSet json definition in a file
+  shell:
+    oc get 'machineset/{{ oc_get_machinesets.stdout_lines[0] }}'
+       -n openshift-machine-api
+       -o json
+       > {{ artifact_extra_logs_dir }}/source_machineset.json
+
+- name: Update the MachineSet with the requested instance type ({{ machineset_instance_type }}) and store in a file
+  shell:
+    "{{ reconfigure_machineset }}
+      {{ artifact_extra_logs_dir }}/source_machineset.json
+      {{ machineset_instance_type }}
```

Comment:
could you add 
```
# The instance type to user for the machineset being scaled
machineset_instance_type: ""
```
to `roles/ensure_machineset/defaults/main/config.yml`, so that the role defines its configuration variables?